src/main/java/com/sprints/interceptors/DisallowUndeclaredRequestParams.java

```diff
@@ -0,0 +1,12 @@
+package com.sprints.interceptors;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface DisallowUndeclaredRequestParams {
```

Comment:
What is the objective of this class  ?

Reply:
It´s the interface I used for the Interceptor for undeclared params on Get operation filters